testing/tests/DevExpress.ui.widgets.editors/textEditorParts/mask.tests.js

```diff
@@ -418,6 +419,48 @@ QUnit.module('typing', moduleConfig, () => {
         assert.equal(handler.callCount, 2, '\'change\' event is not fired after focus out without value change');
         clock.restore();
     });
+
+    QUnit.test('TextEditor with mask option should work correctly with autofill (T869537)', function(assert) {
+        if(!browser.webkit && !(browser.msie && browser.version > 11)) {
+            assert.expect(0);
+            return;
+        }
+
+        const clock = sinon.useFakeTimers();
+        const testText = '555555';
```

Comment:
Let's add try...finally to avoid test dependency.

Reply:
Move up `try` here:
 https://github.com/DevExpress/DevExtreme/pull/12345/files#diff-711ce6fe804895913506b32038e2beedR430